bokeh/command/subcommands/serve.py

```diff
@@ -482,6 +484,13 @@ class Serve(Subcommand):
             type=int,
         )),
 
+        ('--dev', dict(
+            action='store_true',
+            help="Enable live reloading during app development."
+                 "NOTE: This setting has effect ONLY for Tornado>=5.0"
```

Comment:
I think I may have just missed this module previously, it looks like `tornado.autoreload` is in Tornado 4.x? https://www.tornadoweb.org/en/branch4.5/autoreload.html If so, this line about >=5.0 can be removed.